libraries/chain/webassembly/runtimes/eos-vm.cpp

```diff
@@ -85,7 +85,19 @@ void validate(const bytes& code, const whitelisted_intrinsics_type& intrinsics)
    }
 }
 
-void validate( const controller& control, const bytes& code, const wasm_config& cfg, const whitelisted_intrinsics_type& intrinsics ) {
+static bool module_has_valid_sync_call(module& mod) {
```

Comment:
Can this take a `const module& mod` ?

Reply:
It cannot take a `const`, as `get_exported_function()` is not defined as a `const` method.